CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Bugfixes:
 - Make ProxyUtil.IsAccessible(MethodBase) take into account declaring type's accessibility so it doesn't report false negatives for e.g. public methods in inaccessible classes. (@stakx, #289)
+- Fix System.InvalidCastException during IChangeProxyTarget.ChangeProxyTarget method call (@yallie, #293)
```

Comment:
Sorry for missing this last time it would be good to include a bit more context for a reader, could you change it to:
"Fix InvalidCastException calling IChangeProxyTarget.ChangeProxyTarget proxying generic interfaces"